app/angular/src/server/angular-read-workspace.ts

```diff
@@ -24,6 +24,7 @@ export const readAngularWorkspaceConfig = async (
 
     const nxWorkspace = require('@nrwl/workspace').readWorkspaceConfig({
       format: 'angularCli',
+      path: dirToSearch,
```

Comment:
Now that this method is used within a Nx workspace, it began to resolve the `workspace.json` at the root of _this repo_ in unit tests. 

This should not affect end users of Storybook. This is a fix for some tests that use this method.